benchmarks/config/base/base.json

```diff
@@ -133,5 +133,39 @@
       "flags": [ "-n", "100"],
       "extensions": [ "(avx2|asimd)" ]
     }
+  }},
+  {
+  "gemm_models_vector_kernel": {
```

Comment:
move these inside the block `gemm_models` above and call the runs:
* `fp32_3x1024_const_vector`
* `fp32_3x1024_args_vector`

Same below for MLP.